package.json

```diff
@@ -240,7 +240,8 @@
     "@metamask/network-controller@npm:^18.0.1": "patch:@metamask/network-controller@npm%3A18.1.0#~/.yarn/patches/@metamask-network-controller-npm-18.1.0-680908c29a.patch",
     "@metamask/network-controller@npm:^17.2.1": "patch:@metamask/network-controller@npm%3A18.1.0#~/.yarn/patches/@metamask-network-controller-npm-18.1.0-680908c29a.patch",
     "@metamask/network-controller@npm:^18.1.0": "patch:@metamask/network-controller@npm%3A18.1.0#~/.yarn/patches/@metamask-network-controller-npm-18.1.0-680908c29a.patch",
-    "@metamask/keyring-controller@npm:^13.0.0": "patch:@metamask/keyring-controller@npm%3A13.0.0#~/.yarn/patches/@metamask-keyring-controller-npm-13.0.0-d94816a680.patch"
+    "@metamask/keyring-controller@npm:^13.0.0": "patch:@metamask/keyring-controller@npm%3A13.0.0#~/.yarn/patches/@metamask-keyring-controller-npm-13.0.0-d94816a680.patch",
+    "@spruceid/siwe-parser@npm:1.1.3": "2.1.0"
```

Comment:
https://app.circleci.com/pipelines/github/MetaMask/metamask-extension/77541/workflows/f3e0d8ba-02f5-4981-8a8c-7bad6629c47b/jobs/2690272

<img width="1521" alt="Screenshot 2024-04-22 at 11 45 24 PM" src="https://github.com/MetaMask/metamask-extension/assets/20778143/63c78901-3ee6-4fb4-83aa-4796e06a5c24">

hm.. unsure how to handle this lavamoat request. does someone have a suggestion? 
cc: @legobeat @NicholasEllul 

current main/policy.json:
![Screenshot 2024-04-22 at 11 53 05 PM](https://github.com/MetaMask/metamask-extension/assets/20778143/a88d16e0-1918-463a-83b8-ea300c603726)

Reply:
hm. I thought I tried `rm -rf node_modules && yarn install` before `yarn lavamoat:auto` earlier. possibly not. 🤔  I did rerun these commands again with different outputs following our updates.  think it'd be a good idea to run `@ metamaskbot update-policies` again @legobeat?

Reply:
ty! 🙏🏼